### PR TITLE
Update BMLacq756.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I updated the record of BMLacq756 according to Marrassinis catalogue description. 

I found seven msItems with different prayers against illnesses and demon. These prayers appear in similar but not identical form in other manuscripts and may already have their own ID already. 

I am puzzled, whether or not these protective prayers here should get their own new ID or whether we should use the existing IDs for them and described the issue in detail in [#2447](https://github.com/BetaMasaheft/Documentation/issues/2447).

### CODE REVIEW

File: FlorenceBML/BMLacq756/BMLacq756.xml [hunk 1]
```diff
@@ -29,7 +29,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1a1" to="1a34"/>
+                            <title ref="LIT000000">Prayer against Bāryā, Tǝgrǝtyā and Belsibā</title>
+                            <note>Written mostly in Gǝʿǝz and partly in Amharic with invocations of magical names, mostly unintelligible.</note>
+                            <incipit xml:lang="gez"><gap reason="illegible" unit="lines" quantity="3"/> ጸሎት፡ በእንተ፡ ባርያ፡ ወትግርትያ፡ ቤልሲባ፡</incipit>
+                            <explicit xml:lang="gez">ለሰይጣና<gap reason="ellipsis" unit="chars" quantity="2"/> 
+                                አስማቲከ፡ <gap reason="illegible" unit="chars" quantity="4"/> ለአመትከ፡ </explicit>
```
Comment (thea-m) [inline FlorenceBML/BMLacq756/BMLacq756.xml:40]: add `<textLang mainLang="gez" otherLangs="am"></textLang>` (also below, with the correct language)

File: FlorenceBML/BMLacq756/BMLacq756.xml [hunk 2]
```diff
@@ -29,7 +29,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                <ref type="work" corresp="LIT2295Sayfam"/> and through the six words of the Gospel.</note>
+                            <incipit xml:lang="gez">ጸለት፡ በእንተ፡ አርጋኤ<gap reason="illegible" unit="chars" quantity="1"/>ም፡ ሶንታም፡ ቃንታም፡ አጵር፡ መጵር፡ አንተ፡
+                                ባርያ፡ ወሸቶላይ፡ ዘታወሕዝ፡ ደመ፡ ወትቀት፡ ሕፃነ፡ ፈቲሃከ፡ ሁር፡ </incipit>
+                            <explicit xml:lang="gez">ወአርግዕ፡ ደማ፡ ለአመትከ፡ 
+                                <hi rend="rubric"><persName ref="PRS14227WalattaSellase">ወልተ፡ ሥላ<supplied reason="lost">ሴ፡</supplied></persName></hi></explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <locus from="1a155" to="1a183"/>
+                            <title ref="LIT000000000">Mark 5,25-29</title>
+                            <note>A passage from <ref type="work" corresp="LIT1882MarkGo"/> about the healing of haemorrhoids.</note>
```
Comment (thea-m) [inline FlorenceBML/BMLacq756/BMLacq756.xml:76]: I think hemorrhage

File: FlorenceBML/BMLacq756/BMLacq756.xml [hunk 3]
```diff
@@ -29,7 +29,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                ለእግአብሔር፡</explicit>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support><material key="parchment"/></support>
+                                <extent>
+                                    <measure unit="leaf">1</measure>
+                                    <note>Composed of three parts (670+460+410 mm) that are sewn together with thin leather strips.</note>
```
Comment (thea-m) [inline FlorenceBML/BMLacq756/BMLacq756.xml:112]: See https://betamasaheft.eu/Guidelines/?id=scrolls for the encoding of the dimensions of each strip

File: FlorenceBML/BMLacq756/BMLacq756.xml [hunk 4]
```diff
@@ -29,7 +29,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                        <handDesc><handNote xml:id="h1" script="Ethiopic">
+                            <ab type="script"/>
+                            <seg type="script">minute and accurate</seg>
+                            <seg type="ink">Black, red</seg>
+                            <seg type="rubrication">incipits, sacred names, name of owner</seg>
+                            <date from="1700" to="1900" cert="low" datingMethod="paleography"/>
+                          </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">Large miniature in black and red, depicting an angel with a sword, surrounded
```
Comment (thea-m) [inline FlorenceBML/BMLacq756/BMLacq756.xml:140]: I believe `<desc>` is necessary inside decoNote

**Review by thea-m — CHANGES_REQUESTED** [review]
Cool, thank you!

File: FlorenceBML/BMLacq756/BMLacq756.xml (hunk 4 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq756/BMLacq756.xml:148]: Done.

File: FlorenceBML/BMLacq756/BMLacq756.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq756/BMLacq756.xml]: Yes.

File: FlorenceBML/BMLacq756/BMLacq756.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq756/BMLacq756.xml:43]: The information is taken from Marrassini and I can confirm, that the first prayer at least has one Amharic word (ነው), however, the incipit and the explicit do not contain any Amharic words.

**Review by thea-m — APPROVED** [review]

File: FlorenceBML/BMLacq756/BMLacq756.xml (hunk 1 above)
Comment (thea-m) [reply FlorenceBML/BMLacq756/BMLacq756.xml:43]: In any case, the element textLang  should be used regardless, even if only Geez is used

Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq756/BMLacq756.xml:43]: I happened to identify features of Amharic in ms_i1, ms_i2 and ms_i7. Is it correct to encode the phrase individually as for the following: 
```
<msItem xml:id="ms_i7">
                            <locus from="1b160" to="1b189"/>
                            <title>Prayer against Šatolāy and Šatolāwit</title>
                            <textLang mainLang="gez" otherLangs="am"></textLang>
                            <note>Prayer against <term key="demon">Šatolāy</term> and <term key="demon">Šatolāwit</term> for healing from 
                                vaginal bleeding and for protection of newborn babies through the invocation of magic names
                                and through the binding of demons with the bonds of the Father, Son and Holy Spirit similar to 
                                <ref type="work" corresp="LIT6845PPSatolay"/>.</note>
                            <incipit xml:lang="gez">ጸሎት፡ በእንተ፡ <term key="demon">ሸቶላይ፡</term> ወ<term key="demon">ሸቶላዊት፡</term> አጋንንት፡ 
                                ዘትቀትሎ፡ ሕፃናተ፡ ወታወሕዙ፡ ደመ፡ አመሐኩክሙ፡ ወአውገዝኩከሙ፡ ወአርሰርኩክሙ፡</incipit>
                            <explicit xml:lang="gez">ደፈር፡ በሎንቃር፡ ፈውስ፡ ወአሕይዋ፡ ለአመትከ፡ 
                                <hi rend="rubric"><persName ref="PRS14227WalattaSellase">ወልተ፡ ሥላሴ፡</persName></hi>እመ፡ አልቦ፡ ነገር፡ ዘይሰአኖ፡ 
                                ለእግአብሔር፡</explicit>
                        </msItem>
```

**Review by eu-genia — APPROVED** [review]

**Pull request merged**